Readme.md

```diff
@@ -106,7 +107,8 @@ fastify.listen(3000, (err) => {
 for a *cache-response-directive* as defined by RFC 2616.
 + `expiresIn` (Default: `undefined`): a value, in seconds, for the *max-age* the
 resource may be cached. When this is set, and `privacy` is not set to `no-cache`,
-then `', max-age=<value>'` will be appended to the `cache-control` header.
+then `', max-age=<value>'` will be appended to the `cache-control` header. (300 seconds = 5 minutes)
++ `serverExpiresIn` (Default: `undefined`): a value, in seconds, for the *max-age* the resource may be cached on the server. When this is set, and `privacy` is set to `public`,  then `', s-maxage=<value>'` will be appended to the `cache-control` header. (2,592,000 seconds = 30 days)
```

Comment:
I tend to prefer the explanation from MDN:
```
The s-maxage response directive also indicates how long the response is fresh for (similar to max-age) — but it is specific to shared caches, and they will ignore max-age when it is present.
```
Shared caches makes more sense as it's mostly used on CDN edges.

Reply:
I tried to integrate that without copying their language. Let me know what you think